packages/platform-server/shims/package.json

```
@@ -0,0 +1,4 @@
+{
+  "name": "@angular/platform-server/shims",
+  "sideEffects": true
```

`sideEffects` flag can only be defined in primary entry-points.

This has no effect here.

```suggestion
  "sideEffects": true
```

---

It seems that other secondary entry-points have this (for example, [platform-browser/animations][1], [router/upgrade][2]), so I suggest we keep it. Besides consistency with other entry-points, I think it is a good indication (to humans) that this entry-point if side-effect-ful.

[1]: https://github.com/angular/angular/blob/956f75f0689c73c8ea6798c8dec5d7ee38948bc7/packages/platform-browser/animations/package.json
[2]: https://github.com/angular/angular/blob/956f75f0689c73c8ea6798c8dec5d7ee38948bc7/packages/router/upgrade/package.json

---

Indeed, although it might give the wrong impression. That this does field does something. I think humans should check the file that the tools use to determine which entry-point is side-effect free or not.

Anyways, leaving it is not a problem. It just can cause confusion. So, feel free to do whatever you feel.